govdelivery/api.py

```diff
@@ -91,11 +95,12 @@ def create_subscriber(self, contact_details, contact_method="email", send_notifi
 
     def get_all_topics(self):
         path = self.translate_path('/api/account/$account_code/topics.xml')
-        return self.call_api(path, "get", response_parser=xml_response_parsers.listed_category_xml_as_dict)
+        return self.call_api(path, "get", response_parser=xml_response_parsers.topic_xml_as_dict)
 
+    # TODO: Rename method to be more precise
```

Comment:
Did you intend to leave this TODO?

Reply:
Yeah. Renaming would break the existing API, so I'm saving that for down the road.